.pre-commit-config.yaml

```diff
@@ -1,17 +1,19 @@
-default_language_version:
-  python: python3.9
 default_stages: [commit, push]
 fail_fast: false
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v3.4.0
     hooks:
       - id: mixed-line-ending
-        args: ['--fix=lf']
```

Comment:
This seemed too unnecessary and causes problems. While this hook by default it changes to one most used, so in my case it is CRLF and git converts to LF while committing for me.